etc/deb/preinst

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -e
+
+if ! getent passwd tikv >/dev/null; then
+        adduser --quiet --system --no-create-home --home /etc/tikv --shell /usr/sbin/nologin tikv
```

Comment:
should we put home dir in /etc/tikv?. maybe /home/tikv?

Reply:
I think that's not good packaging practice, and we agreed we'd try our best to follow best practices on TiKV/PD packages. 

Commonly packaged applications will use a location like:

*  `/usr/local/$SERVICE`
* `/var/lib/$SERVICE`

Similarly, this is why the configuration is in `/etc/tikv/config.toml`

Ref:

* PSQL https://www.postgresql.org/docs/9.1/creating-cluster.html
* MySQL https://unix.stackexchange.com/a/113983
* Elasticsearch https://www.elastic.co/guide/en/elasticsearch/reference/2.0/setup-dir-layout.html#default-paths
* Docker: https://docs.docker.com/storage/storagedriver/#the-copy-on-write-cow-strategy
* Mailing list: https://www.redhat.com/archives/rpm-list/2005-November/msg00107.html

Reply:
Yes, I recommend to use `/var/lib/tikv` as home directory.